lang_python.adoc

```diff
@@ -103,9 +129,14 @@ available out of the box in standard libraries since Python v3.4, or
 since v2.4 using https://pypi.python.org/pypi/enum34[enum34] PyPi
 compatibility layer.
 
-* On Debian / Ubuntu, this library can be installed with `sudo apt-get
-  install python-enum34`
-* Otherwise, one can use just `sudo pip install enum34`
+* One usually doesn't need to install this lib manually,
+   `pip` will install it automatically if it is needed.
```

Comment:
Is that actually the case? AFAIK, kaitai_struct_python_runtime doesn't depend on `enum34`, so there's no reason why it would be installed automatically.

Reply:
Damn, I have forgotten to make a PR about it.